drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/fakes/KogitoAsyncPackageDataModelOracle.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.screens.scenariosimulation.kogito.client.fakes;
+
+import java.util.List;
+
+import org.kie.soup.project.datamodel.oracle.ModelField;
+import org.kie.workbench.common.services.datamodel.model.PackageDataModelOracleBaselinePayload;
+import org.uberfire.backend.vfs.Path;
+
+/**
+ * Interface provide to emulate the "Oracle" implementation used inside Business Central
+ */
+public interface KogitoAsyncPackageDataModelOracle {
```

Comment:
Can you please verify this interface? There is no implementation here

Reply:
@danielezonca Used in `KogitoRuntimeAsyncPackageDataModelOracle` and `KogitoTestingAsyncPackageDataModelOracle`, in `runtime` and `testing` modules.